.github/workflows/apt.yml

```
@@ -46,6 +47,13 @@ jobs:
       - name: Build deb with Docker
         run: |
           rake apt:build APT_TARGETS=${{ matrix.rake-job }}
+      - name: Build v6 deb with Docker
+        if: ${{ matrix.upgrade_to }} == 6
```

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idif
> When you use expressions in an if conditional, you can, optionally, omit the ${{ }} expression syntax because GitHub Actions automatically evaluates the if conditional as an expression.

```suggestion
        if: matrix.upgrade_to == 6
```

Or

```suggestion
        if: ${{ matrix.upgrade_to == 6 }}
```

---

Thanks!